CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 3.0.0
```

Comment:
No need to bump MAJOR – it's just a minor change, allowing to pass amounts with smaller amount than 0.01.

In other words – nothing to change in application code when we want to update library from 2.x to 3.x.